src/common/test_helper.rs

```diff
@@ -12,15 +12,13 @@
 //! Contains a few helper functions intended purely for tests.
 //! Not intended to be used outside of this crate.
 
-use crate::common::cbor_map::ToCborMap;
-use crate::error::CoseCipherError;
-use crate::token::CoseCipherCommon;
-use crate::{CoseEncrypt0Cipher, CoseMac0Cipher, CoseSign1Cipher};
-use ciborium::value::Value;
 use core::convert::identity;
-use core::fmt::Debug;
-use coset::iana::Algorithm;
-use coset::{Header, Label};
+use core::fmt::{Debug, Display};
+
+use ciborium::value::Value;
+use coset::iana::{Algorithm, SymmetricKeyParameter};
```

Comment:
Are the "unused import" warnings false positives here?

Reply:
They are true positives, I missed the clippy warnings for the test code, thank you for pointing it out. Fixed in 163381e52f885c4f10e100034a8e1e79e50bbfd8.